hub/core/chunk_engine.py

```diff
@@ -921,7 +926,7 @@ def _update_with_operator(
         dt, ht = tensor_meta.dtype, tensor_meta.htype
         samples = intelligent_cast(samples, dt, ht)
         getattr(view, operator)(samples)
-        self._update(index, arr)
+        self._update(index1, arr)
```

Comment:
why it is index1?

Reply:
Full index not required as we already resolved the rest of the index.